src/explore-education-statistics-common/src/modules/table-tool/components/FixedMultiHeaderDataTable.tsx

```diff
@@ -14,11 +14,12 @@ interface Props extends OmitStrict<MultiHeaderTableProps, 'ariaLabelledBy'> {
   innerRef?: Ref<HTMLElement>;
   footnotes?: FullTableMeta['footnotes'];
   source?: string;
+  hiddenText?: string;
```

Comment:
Would call this `footnotesHeadingHiddenText` to avoid ambiguity about what this hidden text actually relates to